src/Products/Viewer/Controllers/ViewerApiController.cs

```diff
@@ -83,54 +64,69 @@ public ViewerConfiguration LoadConfig()
         [Route("loadFileTree")]
         public HttpResponseMessage loadFileTree(PostedDataEntity postedData)
         {
-            string relDirPath = "";
-            // get posted data
-            if (postedData != null)
+            try
             {
-                relDirPath = postedData.path;
+                List<FileDescriptionEntity> filesList = new List<FileDescriptionEntity>();
+                if (!string.IsNullOrEmpty(globalConfiguration.Viewer.GetFilesDirectory()))
+                {
+                    filesList = this.LoadFiles();
+                }
+                return Request.CreateResponse(HttpStatusCode.OK, filesList);
             }
-            // get file list from storage path
-            FileListOptions fileListOptions = new FileListOptions(relDirPath);
-            // get temp directory name
-            string tempDirectoryName = new ViewerConfig().CacheFolderName;
-            try
+            catch (Exception ex)
+            {
+                return Request.CreateResponse(HttpStatusCode.OK, new Resources().GenerateException(ex));
+            }
+        }
+
+        /// <summary>
+        /// Load documents
+        /// </summary>
+        /// <returns>List[FileDescriptionEntity]</returns>
+        public List<FileDescriptionEntity> LoadFiles()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make 'LoadFiles' a static method.](https://app.codacy.com/gh/groupdocs-viewer/GroupDocs.Viewer-for-.NET-MVC/pullRequest?prid=5242614)